src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -178,6 +178,11 @@ private Version getServerVersion() throws IOException {
       LOG.warn("Couldn't get Elasticsearch version, result is null");
       return defaultVersion;
     }
+    if (!result.has("nodes")) {
+      LOG.error("Couldn't get Elasticsearch version (result is has no nodes); "
+              + "assuming {}, result: {}", defaultVersion, result);
```

Comment:
```suggestion
      LOG.warn("Couldn't get Elasticsearch version from result {} (result has no nodes). Assuming {}.", result, defaultVersion);
```
nit on log level and message